Lib/zipfile.py

```diff
@@ -1583,7 +1583,7 @@ def extractall(self, path=None, members=None, pwd=None):
            by namelist().
         """
```

Comment:
Does the doc needs an update?

Reply:
You're right. It should be `infolist` there instead of `namelist`.